### DISCUSSION TITLE
Failed to load component "544c1372-7e5a-49ec-8db6-812f76c375f2" (pnpSearchBoxWebPart). Original error: Failed to load URL 'https://localhost:4321/dist/SearchBoxWebPartStrings_en-us.js'

### DISCUSSION THREAD

**Discussion by levalencia** [microsoft-search/pnp-modern-search#1829] [Q&A] [answered]
Hello Team

After 9 months (lol) my company approved finally to use an Open Source Package  in our production tenant (Modern Search), however they told me that I had to rename it to comply with their naming conventions  (webpart names and package)

I downloaded the source code for: 4.5.3.

I followed this article to rename the webpart names and the package name:
https://www.techrobbers.com/2021/10/how-to-update-webpart-name-package-name.html

I ran npm install

Then I created the package with :
gulp package-solution

And send them the package for deployment on tenant level.

After adding the APP on the site collection and adding the Search Box Webpart we get this error:

```
Something went wrong
If the problem persists, contact the site administrator and give them the information in Technical Details.
TECHNICAL DETAILS
[SPLoaderError.loadComponentError]:
Failed to load component "544c1372-7e5a-49ec-8db6-812f76c375f2" (pnpSearchBoxWebPart). Original error: Failed to load URL 'https://localhost:4321/dist/SearchBoxWebPartStrings_en-us.js' for resource 'SearchBoxWebPartStrings' in component '544c1372-7e5a-49ec-8db6-812f76c375f2' (pnpSearchBoxWebPart). There was a network problem. Make sure that you are running 'gulp serve' and you have run 'gulp trust-dev-cert'.

INNERERROR:
Failed to load URL 'https://localhost:4321/dist/SearchBoxWebPartStrings_en-us.js' for resource 'SearchBoxWebPartStrings' in component '544c1372-7e5a-49ec-8db6-812f76c375f2' (pnpSearchBoxWebPart). There was a network problem. Make sure that you are running 'gulp serve' and you have run 'gulp trust-dev-cert'.
CALLSTACK:
Error
    at t [as constructor] (https://modernb.akamai.odsp.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_9b319f7cf8029bc1f884c55daf0ceb7d.js:70:214302)
    at new t (https://modernb.akamai.odsp.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_9b319f7cf8029bc1f884c55daf0ceb7d.js:71:9104)
    at Function.e.buildErrorWithVerboseLog (https://modernb.akamai.odsp.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_9b319f7cf8029bc1f884c55daf0ceb7d.js:71:46185)
    at Function.e.buildLoadComponentError (https://modernb.akamai.odsp.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_9b319f7cf8029bc1f884c55daf0ceb7d.js:71:42302)
    at https://modernb.akamai.odsp.cdn.office.net/files/sp-client/chunk.systemjs-component-loader_en-us_c4c3fed43a2e1075fc65.js:1:60176
```

I am assuming that I did something wrong when packaging? It looks like I packaged for dev no production and thats why its searching resources on localhost?

Any tips please?

**Comment by FranckyC** [comment]
Hi @levalencia, your package points to localhost. Use the `gulp package-solution --ship` instead

**Comment by levalencia** [comment]
Thanks a lot FrankyC

I downloaded the latest version and I did:

npm install
gulp build
gulp bundle 
and then
gulp package-solution --ship


Now the error is different.

[SPLoaderError.loadComponentError]:
Failed to load component "544c1372-7e5a-49ec-8db6-812f76c375f2" (pnpSearchBoxWebPart). Original error: Failed to load URL 'https://ourtenant.sharepoint.com/sites/OurSiteName/ClientSideAssets/59903278-dd5d-4e9e-bef6-562aae716b8b/SearchBoxWebPartStrings_en-us.js' for resource 'SearchBoxWebPartStrings' in component '544c1372-7e5a-49ec-8db6-812f76c375f2' (pnpSearchBoxWebPart). The file was not found in the server.

**Comment by FranckyC** [comment, marked as answer]
You need also to do `gulp bundle --ship` and `gulp build --ship`.
